tests/components/bond/test_light.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for the Bond light device."""
+from datetime import timedelta
+import logging
+
+from bond import Actions, DeviceTypes
+
+from homeassistant import core
+from homeassistant.components.light import DOMAIN as LIGHT_DOMAIN
+from homeassistant.const import ATTR_ENTITY_ID, SERVICE_TURN_OFF, SERVICE_TURN_ON
+from homeassistant.helpers.entity_registry import EntityRegistry
+from homeassistant.util import utcnow
+
+from .common import setup_platform
+
+from tests.async_mock import patch
+from tests.common import async_fire_time_changed
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def ceiling_fan(name: str):
+    """Create a ceiling fan (that has built-in light) with given name."""
+    return {
+        "name": name,
+        "type": DeviceTypes.CEILING_FAN,
+        "actions": [Actions.TOGGLE_LIGHT],
+    }
+
+
+async def test_entity_registry(hass: core.HomeAssistant):
+    """Tests that the devices are registered in the entity registry."""
+    await setup_platform(hass, LIGHT_DOMAIN, ceiling_fan("name-1"))
+
+    registry: EntityRegistry = await hass.helpers.entity_registry.async_get_registry()
+    assert [key for key in registry.entities.keys()] == ["light.name_1"]
```

Comment:
Iterating a dict iterates the keys by default, we can remove `.keys()`.

Reply:
Thanks, fixed this here: https://github.com/home-assistant/core/pull/37821